src/sentry/services/hybrid_cloud/user/impl.py

```diff
@@ -180,13 +184,13 @@ def get_first_superuser(self) -> RpcUser | None:
             return None
         return serialize_rpc_user(user)
 
-    def get_or_create_user_by_email(
+    def get_or_create_user_by_email_v2(
         self, *, email: str, ident: str | None = None, referrer: str | None = None
-    ) -> tuple[RpcUser, bool]:
+    ) -> RpcUserCreationResult:
```

Comment:
I think we'd need to keep the old method around as well. During deploys callers will still be using the old method.